flink-runtime/src/main/java/org/apache/flink/runtime/clusterframework/FlinkResourceManager.java

```diff
@@ -113,7 +111,8 @@
 	/** The service to find the right leader JobManager (to support high availability) */
 	private final LeaderRetrievalService leaderRetriever;
 
-	/** The currently registered resources */
+	/** The resources which have been registered at the ResourceManager. This indicates the set
+	 * of containers where a TaskManager has been started and registered at a JobManager.  */
```

Comment:
Yes as you've said it in another comment, it does not represent TMs which are currently registered but TMs which are started and have at least once been registered. For the sake of simplicity, we could rename the field into `startedWorkers`.